### ISSUE TITLE
replace copied and pasted code of pyliczba

### ISSUE THREAD

**Issue by sq6jnx** [sq6jnx/sr0wx.py#9]
`pl_google/pl_google.py` contains lots of code copied and pasted form dowgird's pyliczba. It should be replaced with:
 - either git submodule to my fork of pyliczba (https://github.com/sq6jnx/pyliczba)
 - either used by `import pyliczba`, but pyliczba is not pip-compatible

**Comment by sq6jnx** [comment]
One thing to mention: in current state tests run in Python2.7 pass, but in 3.4 there's a import problem.

@wprzyb, mayke you can make a `pip`-like package for pyliczba? What do you think?

**Comment by wprzyb** [comment]
@sq6jnx 

https://github.com/wprzyb/pyliczba/tree/pip-pyliczba - it looks like working, i think i need to test that a bit, but problem i have now is that after you install pyliczba from pip (now from file, but by pip, i didn't register that yet) the way of using functions is:

```python
from pyliczba import kwotaslownie
x = 10
test = kwotaslownie.kwotaslownie(x)
```

So tests written without doubling the kwotaslownie (once for module and once for function) in code don't work. I don't really know by now how to deal with this to cut one 'kwotaslownie' from that function call.

**Comment by wprzyb** [comment]
```python
from pyliczba.kwotaslownie import kwotaslownie 
```
Does the job. Resolved?

**Comment by sq6jnx** [comment]
This is not the repo we should discuss in. Please create MR in pyliczba.

Hint: add `__init__.py` file with from `pyliczba import *` and rename `kwotaslownie` to `pyliczba`

**Comment by sq6jnx** [comment]
Problem solved in bfdaa955ca30eeef769b92a946fc9fd67cdbecf3.

@wprzyb, @wiget: thank you both for your help.
